app/js/components/manageApps/Addon.jsx

```diff
@@ -222,10 +253,11 @@ class Addon extends Component {
         <button
           type="button"
           className="btn btn-danger btn-delete"
-          onClick={app.uuid ? this.handleModuleDelete : this.handleOwaDelete}
+          onClick={app.uuid? this.handleModuleDelete : this.handleOwaDelete}
```

Comment:
Since `handleModuleDelete` and `handleOwaDelete` are the same they should be replaced with `handleAddonDelete` hence:
`onClick={this.handleAddonDelete}`

Reply:
Removes the unnecessary conditional statement and repeated functionality